### PR TITLE
putting a link to travis results at the top

### PR DESCRIPTION
**Pull request by lbergelson**
I'm sick of scrolling to the bottom to find the test report uri.
This saves the pain of having to scroll all the way down.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -52,6 +52,8 @@ cache:
     - $HOME/gcloud/
     - $HOME/site-library
 before_install:
+- REPORT_PATH=${TRAVIS_BRANCH}_$T{RAVIS_JOB_NUMBER};
```
Comment (jamesemery) [inline .travis.yml]: $T{RAVIS_JOB_NUMBER} -> ${TRAVIS_JOB_NUMBER}

Comment (jamesemery) [reply .travis.yml]: Also perhaps we should hide this text behind $TRAVIS_SECURE_ENV_VARS == true  so its not pointing to some non-existant bucket if someone external submits a PR.

Comment (lbergelson) [reply .travis.yml]: That's a a good idea.

Comment (lbergelson) [reply .travis.yml]: oh, and a good typo catch, this is why I should never make "minor changes" after checking that something is working....

File: src/main/java/org/broadinstitute/hellbender/utils/downsampling/AlleleBiasedDownsamplingUtils.java [hunk 2]
```diff
@@ -6,11 +6,8 @@
 import org.apache.commons.collections4.map.DefaultedMap;
 import org.apache.logging.log4j.Logger;
 import org.broadinstitute.hellbender.exceptions.UserException;
-import org.broadinstitute.hellbender.utils.BaseUtils;
```
Comment (droazen) [inline src/main/java/org/broadinstitute/hellbender/utils/downsampling/AlleleBiasedDownsamplingUtils.java]: You  might want to clean up the diff here though...

File: .travis.yml [hunk 3]
```diff
@@ -52,6 +52,10 @@ cache:
     - $HOME/gcloud/
     - $HOME/site-library
 before_install:
+- REPORT_PATH=${TRAVIS_BRANCH}_${TRAVIS_JOB_NUMBER};
+- if [ $TRAVIS_SECURE_ENV_VARS == true && $TRAVIS_EVENT_TYPE != cron ]; then
```
Comment (jamesemery) [inline .travis.yml]: It looks like you may have needed a second set of brackets. If you check the travis logs this line is currently erroring with the following message and failing to print the path correctly: 
`0.01s$ if [ $TRAVIS_SECURE_ENV_VARS == true && $TRAVIS_EVENT_TYPE != cron ]; then echo "Test report will be written to  https://storage.googleapis.com${HELLBENDER_TEST_LOGS}${REPORT_PATH}/tests/test/index.html"; fi
/home/travis/.travis/job_stages: line 57: [: missing `]'`

**Comment by jamesemery** [comment]
The most recent run seems to have fixed it, 👍

**Pull request merged**